0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -129,6 +129,8 @@ The response body is a JSON object that includes basic account details necessary
 | `receiver_info` | Object | Arbitrary additional information about the receiver. This field has no schema and the receiver may include any fields they choose. The field names listed below are recommended merely for interoperability purposes. |
 | `receiver_info.name` | String | _(OPTIONAL)_ Full name of the individual, company or organization the receiver represents |
 | `receiver_info.image_url` | HTTPS URL | _(OPTIONAL)_ URL where the sender can get a picture representation of the receiver |
+| `signature_info.algorithm` | String | _(OPTIONAL)_ Algorithm that will be used to sign ILP errors and fulfillment data |
+| `signature_info.public_key` | String | _(OPTIONAL)_ Public key that will be used to sign ILP errors and fulfillment data |
```

Comment:
It should say: _Public key that will be used to **verify** signatures ..._

Better not to publish the signing key ;-)